root/static/scripts/tests/release-editor/fields.js

```diff
@@ -186,52 +186,22 @@ fieldTest("tracks are set correctly when the cdtoc is changed", function (t, rel
 
 fieldTest("Tracks' time are changed correctly when inputting values in the medium tracklist editing form tab", function (t, release){
     t.plan(8);
-    
-    var lengths = [
-        "5",
-        "69",
-        "174",
-        "6000",
-        "7400",
-        "10000",
-        "96900",
-        "3723494",
-    ];
 
-    var output = [
-        "0:05",
-        "1:09",
-        "2:54",
-        "1:00:00",
-        "1:14:00",
-        "1:00:00",
-        "10:09:00",
-        "?:??",
+    const tests = [
+        {input: "5", output: "0:05"},
+        {input: "69", output: "1:09"},
+        {input: "174", output: "2:54"},
+        {input: "6000", output: "1:00:00"},
+        {input: "7400", output: "1:14:00"},
+        {input: "10000", output: "1:00:00"},
+        {input: "96900", output: "10:09:00"},
+        {input: "3723494", output: "?:??"}
     ];
 
-    var medium = new fields.Medium({ tracks: [ {} ] }, release);
```

Comment:
You should not have removed this line, as it is needed to define `medium` for the rest of the tests.

Reply:
definitely should not have

Reply:
restored in https://github.com/metabrainz/musicbrainz-server/pull/750/commits/b7cf778c7cd7e799107e10f51983c1db2275f28d